### PR TITLE
One Click Buy: Add option

### PR DESCRIPTION
**Pull request by allangrds**
## Description
Before test this PR, review that https://github.com/pagarme/artis/pull/240

Use pagarme-core branch `add/create-card-with-ek`

### How to test
Insert param `cardId` inside `configs`.

Closes https://github.com/pagarme/mercurio/issues/217

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

File: src/utils/strategies/pagarme.js [hunk 1]
```diff
@@ -278,16 +281,23 @@ const request = (data) => {
...
   const amount = propOr(0, 'amount', transaction)
+  const cardId = path(['creditCard', 'cardId'], apiData)
 
   const installments = pathOr({}, [
     'paymentConfig',
     'creditcard',
     'installments',
   ], transaction)
 
+  const willGetInstallments = ifElse(
```
Comment (thiagommedeiros) [inline src/utils/strategies/pagarme.js]: Não seria melhor `shouldGetInstallments`?

**Comment by vcapretz** [comment]
pra esse PR aqui precisamos que o outro seja margeado, certo? vou esperar então pra revisar!

**Comment by vcapretz** [comment]
precisa do rebase com master agora, vcs chegaram a ver meu comentário em cima @allangrds @thiagommedeiros?

**Comment by augusto-jm-amaral** [comment]
@allangrds faz um rebase para eu aprovar :)

**Comment by allangrds** [comment]
@vcapretz @augusto-jm-amaral Rebase feito

**Pull request merged**